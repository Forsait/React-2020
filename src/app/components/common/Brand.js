// @flow
import React from 'react';
// $FlowFixMe
import { createUseStyles } from 'react-jss';

const useStyles = createUseStyles({
  brand: {
    color: '#F65261',
    fontSize: '20px',
    fontWeight: 500,
    letterSpacing: 0,
    lineHeight: '24px',
  },
});

type Props = {className?: string};

export default function Brand(props: Props) {
  const classes = useStyles();
  return (
    <div className={`brand ${props.className || ''}  ${classes.brand}`}>
      <span style={{ fontWeight: 800 }}>netflix</span>
      roulette
    </div>
  );
}
