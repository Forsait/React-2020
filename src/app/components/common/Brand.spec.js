import React from 'react';
import { shallow } from 'enzyme';

import Brand from './Brand';

describe('Brand component', () => {
  it('should be render correctly', () => {
    const com = shallow(<Brand />);
    expect(com).toMatchSnapshot();
  });
});
