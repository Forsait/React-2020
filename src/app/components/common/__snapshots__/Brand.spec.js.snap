// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand component should be render correctly 1`] = `
<div
  className="  undefined"
>
  <span
    style={
      Object {
        "fontWeight": 800,
      }
    }
  >
    netflix
  </span>
  roulette
</div>
`;
